jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    clearMocks: true,
+    moduleFileExtensions: ['js', 'ts'],
+    testMatch: ['**/*.test.ts'],
+    transform: {
+      '^.+\\.ts$': 'ts-jest'
+    },
+    verbose: true
+  }
```

Comment:
Something weird happened with the indentation here, could you take a look? This is all in a mix of 2-spaced and 4-spaced indentation, and the last line is unnecessarily indented.